search.go

```diff
@@ -33,6 +34,8 @@ func init() {
 	libTopMatch = regexp.MustCompile(`^([A-Za-z$_][A-Za-z0-9$_:.]*?)\/([A-Za-z$_A-Za-z0-9$_]*?)$`)
 	topPropMatch = regexp.MustCompile(`^([A-Za-z$_A-Za-z0-9$_]*?)\.([A-Za-z$_<+|[>\/^~&*%=\-][A-Za-z0-9$_\]=\/<>\-]*?)$`)
 	libTopPropMatch = regexp.MustCompile(`^([A-Za-z$_][A-Za-z0-9$_:.]*?)\/([A-Za-z$_A-Za-z0-9$_]*?)\.([A-Za-z$_<+|[>\/^~&*%=\-][A-Za-z0-9$_\]=\/<>\-]*?)$`)
+	// Block of code match.
+	blockCodeMatch = regexp.MustCompile("```.*```")
```

Comment:
Check for the presence of an escaping backslash before the start of the first triple backticks to catch escaped blocks

Reply:
Addressed in the next commit with the agreed upon regex.